src/components/chat/chat-thread.test.tsx

```diff
@@ -0,0 +1,90 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { ModalProvider } from "react-modal-hook";
+import { ENavTab } from "../../models/view/nav-tabs";
+import { ChatThread } from "./chat-thread";
+import { UserModelType } from "src/models/stores/user";
+
+jest.mock("../../hooks/use-stores", () => ({
+  useUIStore: () => ({
+    showChatPanel: true,
+    selectedTileIds: []
+  }),
+  useTypeOfTileInDocumentOrCurriculum: () => "Text",
+}));
+
+const makeFakeCommentThread = (title: string, tileId: string, uid: string) => {
+    return {
+        title,
```

Comment:
Two-space indent
```suggestion
const makeFakeCommentThread = (title: string, tileId: string, uid: string) => {
  return {
    title,
```
etc.